### PR TITLE
Changed/added error messages

### LINKED ISSUE #39: Error message for stacking repetition cards
Currently, if you try to stack repetition cards, the same error message is given about not being able to do so as there is no instruction or group card. Let's detect this case and provide a more meaningful error message.

### PR DESCRIPTION
**Pull request by vincentcote87**
I have changed some of the error messages to be more relevant to the particular move a player is attempting. I have also added error messages for the hack and group card in case a player tries to add them to a stack. @johnanvik please read over the messages carefully and advise if the text needs to be rewritten.

fix #39

### CODE REVIEW

File: src/components/Stack.vue [hunk 1]
```diff
@@ -335,6 +340,15 @@ export default {
 
               break;
 
+          case 'H':
+            $('button[stackId="'+this.stackId+'"]').attr("data-content", "Hack cards are not played on a stack, you must open the opponent's stack to hack them." );
+            $('button[stackId="'+this.stackId+'"]').popover('toggle');
+            break;
+
+          case 'G':
+            $('button[stackId="'+this.stackId+'"]').attr("data-content", "Group cards are not played on a stack, click the checkbox(s) to group the cards together (the combined value of the stack(s) must equal the value of the group card)." );
```
Comment (johnanvik) [inline src/components/Stack.vue]: "...stack. Click the checkbox(es) to select the cards to group together. The total of the selected card(s) must equal the value of the group card."

File: src/components/Stack.vue [hunk 2]
```diff
@@ -297,7 +302,7 @@ export default {
             console.log('current active card: ' + this.$store.getters.getActiveCard)
           if (thisStack.cards.length === 0) {
 
-                $('button[stackId="'+this.stackId+'"]').attr("data-content", "You cannot add a variable card to a stack without an instruction card. Try adding that card to a stack with an instruction card." );
+                $('button[stackId="'+this.stackId+'"]').attr("data-content", "You can only add variable cards to a stack with an open variable (Rx) repetition card or an existing variable card." );
```
Comment (vincentcote87) [inline src/components/Stack.vue]: I believe so, I was able to replace a v2 with a v4. Let me know if you want to keep that feature or not.

Comment (johnanvik) [inline src/components/Stack.vue]: OK. I didn't have a running version handy to check. Keep this functionality.

**Comment by vincentcote87** [comment]
I have updated the lines as requested, @johnanvik  let me know if thats what you had in mind.

**Comment by johnanvik** [comment]
Looks like these changes conflict with the master now (guessing from recent merge). Please resolve this.

**Comment by vincentcote87** [comment]
Merge conflicts have been resolved.

**Pull request merged**